spec/yaml/swiftnav/sbp/signing.yaml

```diff
@@ -54,3 +56,35 @@ definitions:
             type: array
             fill: u8
             desc: ED25519 certificate bytes.
+
+  - MSG_ED25519_SIGNATURE:
```

Comment:
These changes will need to be reflected [here](https://github.com/swift-nav/rtcm2signed/blob/f05b2ca3a4b177dd3d63f876e9220855ea98ca6b/rtcm2signed/rtcm/frame_signer.go#L22-L29) right?

Reply:
Yeah, although we are pretty close to moving the signing from there, so I don't think we need to update that